common/src/main/java/com/linecorp/centraldogma/internal/jsonpatch/PathValueOperation.java

```diff
@@ -66,6 +66,10 @@
         this.value = value.deepCopy();
     }
 
+    public JsonNode value() {
```

Comment:
This seems unused?

Reply:
It's used in the `MetadataApiService` like the following:

```
final ProjectRole role = ProjectRole.of(operation.value());
```